sources/HUBViewControllerImplementation.m

```diff
@@ -48,6 +48,8 @@
 #import "HUBActionHandler.h"
 #import "HUBViewModelDiff.h"
 
+CFTimeInterval const kDownloadingTimeTreshold = 0.07;
```

Comment:
Please rename it to `HUBDownloadTimeThreshold` to adhere to our style guidelines:

Only pure C constants used by pure C code should be prefixed with a `k`. All other constant, and symbols, should follow normal Cocoa naming guidelines. I.e. use the symbol prefix (`HUB`).